_data/governments.yml

```
@@ -102,6 +102,7 @@ Brazil:
   - secultce
   - sepinf-inc
   - servicosgovbr
+  - smcti Pato Branco
```

The list is just the GitHub handle. The spaces/name will fail on the CI run
```suggestion
  - smcti
```